man/gbz80.7

```diff
@@ -733,7 +770,7 @@ Flags: None affected.
 .Ss JP n16
 Jump to address
 .Ar n16 ;
-effectively, store
+effectively, copy
```

Comment:
'load' for consistency?

Reply:
The whole file consistently uses "copy".

Reply:
Thanks. I just noticed that this was changed from here on.

I read the proposed change of DAA in #1564 and felt that the wording there was somewhat "complicated" by the acronym's full form, so I errorneously assumed that using the origin to the mnemonic was generally intended, but I was wrong. E.g., it isn't done for the bit shifts either. So that's fine.

Reply:
Usually we use the mnemonic terminology ("complement" for `CPL`, "accumulator" for `A`, etc) but "load" is too ambiguous about which parameter is the destination, so "copy" is preferred.